src/interfaces/filing-interfaces/registration-filing-interface.ts

```diff
@@ -9,7 +9,6 @@ export interface RegistrationIF {
   business: {
     identifier: string
     naics?: NaicsIF
-    natureOfBusiness?: string
```

Comment:
Obsolete. The UI was only setting this to '' in the filing JSON, and the API was ignoring it anyway (confirmed with Lekshmi).